test/crdsvalidation/gateway-operator.konghq.com/controlplane_test.go

```diff
@@ -196,4 +196,164 @@ func TestControlPlane(t *testing.T) {
 			},
 		}.Run(t)
 	})
+
+	t.Run("watch namespaces", func(t *testing.T) {
+		common.TestCasesGroup[*operatorv1beta1.ControlPlane]{
+			{
+				Name: "no watch namespaces",
+				TestObject: &operatorv1beta1.ControlPlane{
+					ObjectMeta: common.CommonObjectMeta,
+					Spec: operatorv1beta1.ControlPlaneSpec{
+						ControlPlaneOptions: operatorv1beta1.ControlPlaneOptions{
+							Deployment: operatorv1beta1.ControlPlaneDeploymentOptions{
+								PodTemplateSpec: &corev1.PodTemplateSpec{
+									Spec: corev1.PodSpec{
+										Containers: []corev1.Container{
+											{
+												Name:  "controller",
+												Image: "kong:3.9.0",
+											},
+										},
+									},
+								},
+							},
+						},
+					},
+				},
+			},
+			{
+				Name: "watch namespaces all",
+				TestObject: &operatorv1beta1.ControlPlane{
+					ObjectMeta: common.CommonObjectMeta,
+					Spec: operatorv1beta1.ControlPlaneSpec{
+						ControlPlaneOptions: operatorv1beta1.ControlPlaneOptions{
+							Deployment: operatorv1beta1.ControlPlaneDeploymentOptions{
+								PodTemplateSpec: &corev1.PodTemplateSpec{
+									Spec: corev1.PodSpec{
+										Containers: []corev1.Container{
+											{
+												Name:  "controller",
+												Image: "kong:3.9.0",
+											},
+										},
+									},
+								},
+							},
+						},
```

Comment:
This doesn't need to be repeated there is `validControlPlaneOptions` defined at the top of this test

Reply:
👍 Done in e269d7f42943233d1e1998090661383b0c76bbb7.